node/peer.js

```diff
@@ -62,7 +63,13 @@ function TChannelPeer(channel, hostPort, options) {
         );
     }
 
-    self.setScoreStrategy(PreferOutgoing);
+    self.preferOutgoing = self.options.preferOutgoing;
```

Comment:
preferOutgoing cannot be the default, since service side should preferIncoming. In general, no preference cannot cause any issue. preferOutgoing & preferIncoming are more like optimization.